qiita_db/user.py

```diff
@@ -126,31 +125,33 @@ def login(cls, email, password):
         IncorrectPasswordError
             Password passed is not correct for user
         """
-        # see if user exists
-        if not cls.exists(email):
-            raise IncorrectEmailError("Email not valid: %s" % email)
-
-        if not validate_password(password):
-            raise IncorrectPasswordError("Password not valid!")
-
-        # pull password out of database
-        conn_handler = SQLConnectionHandler()
-        sql = ("SELECT password, user_level_id FROM qiita.{0} WHERE "
-               "email = %s".format(cls._table))
-        info = conn_handler.execute_fetchone(sql, (email, ))
-
-        # verify user email verification
-        # MAGIC NUMBER 5 = unverified email
```

Comment:
Not your fault, but could you change this to use the new convert_to_id() function?